ml-proto/test/memory.wasm

```diff
@@ -54,8 +66,8 @@
           (break)
         )
         (setlocal 2 (mul.i32 (getlocal 0) (const.i32 4)))
-        (setnears.i32 (getlocal 2) (getlocal 0))
-        (setlocal 1 (getnears.i32 (getlocal 2)))
+        (stores.i32 (getlocal 2) (getlocal 0))
+        (setlocal 1 (loads.i32 (getlocal 2)))
```

Comment:
I think at the AST/spec level we shouldn't have an "unspecified" option, but rather leave this as a text-format-level syntactic sugar (as is the case in this PR) and a binary-format-level encoding optimization.  Of course we'd want tests for the text/binary format, but I think the way we should test this case (and surely others to follow) is to have testing commands that parse a given binary/text string and assert that the resulting AST has the expected values.